packages/ses/src/whitelist.js

```diff
@@ -1532,7 +1539,10 @@ export default {
   CompartmentPrototype: {
     constructor: 'Compartment',
     evaluate: fn,
-    global: getter,
+    globalThis: getter,
+    import: asyncFn,
```

Comment:
This points out that it is not purely an implementation decision whether to use a function or an async function here. The comment in `FunctionInstance` above also reminds me that shimmed-standard methods, whether async or not, need to be defined using concise method syntax, or, if not this sensitive, arrow syntax. They should not be defined using `function` syntax since we don't want a contruct behavior or a `prototype` property.

Whether this method is observably async or not needs to parallel how we want to spec it. Does the current draft spec take a position?

Reply:
The proposal as written suggests that import is expressly an async function. This seems like a good decision since it allows us to express without reservation that the function will return a promise, without necessarily requiring the use of `await`.

```ts
  async import(specifier: string): Promise<{namespace: ModuleNamespace}>;
```